src/preset.js

```diff
@@ -4,29 +4,146 @@
 
 'use strict';
 
+/**
+ * @typedef {{
+ *   fileName: string,
+ *   isGeneRegex: boolean,
+ *   isConditionRegex: boolean,
+ *   geneInput: string,
+ *   conditionInput: string
+ * }}
+ */
+genotet.ExpressionParams;
```

Comment:
Have we defined those already?
https://github.com/ViDA-NYU/genotet/blob/refactor/src/components/expression/expression.js#L16